### DISCUSSION TITLE
可不可以加一个选择office版本的功能？

### DISCUSSION THREAD

**Discussion by xingyeyeyeye** [OdysseusYuan/Mocreak#20] [Q&A] [answered]
我想安装以前的版本。

**Comment by OdysseusYuan** [comment]
感谢您的使用。您提及的“Office 版本选择”是否指 2016~2019的这种大迭代版本？

**Comment by OdysseusYuan** [comment, marked as answer]
您好，Mocreak 已发布新版本，新增支持 Office 2016、2019 版本，您可尝试使用。

下载地址：https://github.com/OdysseusYuan/Mocreak/releases/latest

以上，如有问题，欢迎随时沟通。
